scss/bitstyles.scss

```diff
@@ -77,6 +78,7 @@
 @import 'bitstyles/utilities/items/';
 @import 'bitstyles/utilities/self/';
 @import 'bitstyles/utilities/justify/';
+@import 'bitstyles/utilities/height/';
```

Comment:
would it make sense to eventually sort these imports by name :D After all it is a lot by now

Reply:
I’ve been thinking about that. It’ll take a little investigation — some of these need to come before others to allow certain conventions to hold true (`.u-hidden` can be overridden at specified screensizes using e.g. `.u-block@m` or `.u-flex@l`, but that only works if `hidden` comes first in source order).

I’ll add a ticket for it :)

Reply:
If some need to come before than those can just be sorted at the top and have a space in between, to indicate that they need to come before. But does not need to happen in this ticket :)

Reply:
https://github.com/bitcrowd/bitstyles/pull/452